game-core/src/main/java/games/strategy/engine/lobby/client/ui/LobbyGamePanel.java

```diff
@@ -200,7 +200,6 @@ private void mouseOnGamesList(final MouseEvent e) {
 
   private Collection<Action> getBotAdminGamesListContextActions(final GameDescription description) {
```

Comment:
`description` is now an unused parameter.

Reply:
Fixed in: ddbe1cb1b